### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by brighton0102**
### In this milestone I : 

Insert the following data:

- Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
- Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
- Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
- Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
- Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
- Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.

Write queries to answer the following questions:

- How many animals are there?

animal count

- What is the average weight of animals?

avg weight

- Delete all data and rollback

delete animals

- What is the average number of escape attempts per animal type of those born between 1990 and 2000? 

escp attempts

- What is the minimum and maximum weight of each type of animal?

min max

- Delete and update animal table

negative weights

- Who escapes the most, neutered or not neutered animals?
neutered

- How many animals have never tried to escape?
never tried to escape

- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.

pokemon and digimon

- update the animals table by setting the species column to unspecified 

unspecified

### CODE REVIEW

**Review by skyv26 — COMMENTED** [review]
## Hi @brighton0102, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🟨 



## Reason 🗒️ 
You have submitted a project, that doesn't have the correct README template structure, as per the `general` requirements. You should add the `getting-started` section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -9,5 +8,72 @@ SELECT * FROM animals WHERE neutered = true;
...
+where name like '%mon';
+select * from animals;
+
+/* Update the animals table, setting the species column to 'pokemon' for rows where species is not already set*/
+update animals set species = 'pokemon'
+where species is null;
+select * from animals;
+commit;
+select * from animals;
+
```
Comment (Munsa1) [inline queries.sql]: - [ ] This transaction is missing a begin statement, kindly add one as per the requirement

e.g

```
begin;
update animals set species = 'digimon'
where name like '%mon';
select * from animals;

/* Update the animals table, setting the species column to 'pokemon' for rows where species is not already set*/
update animals set species = 'pokemon'
where species is null;
select * from animals;
commit;
select * from animals;
```

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @brighton0102, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Added Screnshot For Queries :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
### Important :warning: 
- [ ] For proper documentation ensure your queries are are labeled correctly, for example put the question being answered on top of the screenshot answering the question. Right  now your screenshots are jumbled making it dificult to understand what is going on.

Check screencast
[cinnamon-2023-11-23T160048+0200.webm](https://github.com/brighton0102/vet-clinic/assets/81932478/fb87a1b5-9ebf-4208-b903-4dd95eb0ecf8)

Example of how best to document your work
[cinnamon-2023-11-23T160317+0200.webm](https://github.com/brighton0102/vet-clinic/assets/81932478/4901d669-1b07-42ab-b0b6-973f03dfb615)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Munsa1** [comment]
- [ ] I missed this. Kindly add the `Future Features` heading to your readme file for a more professional readme file
image

**Review by topeogunleye — APPROVED** [review]
### Hi @brighton0102, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
1. **Transaction Begin Statement:**
   - [ ] Added the necessary `begin` statement to the transaction as required. This ensures proper transaction handling.

2. **Readme Enhancement:**
   - [ ] Included the "Future Features" heading in your readme file for a more professional and organized presentation of future developments.

These improvements enhance the reliability of your code and contribute to a more polished project documentation. Great attention to detail! Keep up the excellent work! 🚀
 
## [Optional] suggestions:
👉 One optional suggestion for improvement is to follow the convention for naming your branch. Usually, branch names should use the past tense verb to indicate that the feature has been implemented. For example, instead of `update-animals-table`, you could use `updated-animals-table`. This would make your branch name more consistent and clear for future pull requests. 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**